config/locales/en.yml

```diff
@@ -283,7 +283,7 @@ en:
       lede: Emails sent by your form, either to you or to people filling it in, will come from this address
       messages:
         default: You need to change the default no-reply-moj-forms address before your form can go live
-        pending: <strong>Now check the email you set for a validation link</strong><br /><br />The email will come from <b>Amazon Web Services</b>. The link is valid for 24 hours. 
+        pending: <strong>Now check the email you set for a validation link</strong><br /><br />The email will come from <b>Amazon Web Services</b>. The link is valid for 24 hours.
```

Comment:
oh i see, hence the question from @chrispymm about brakeman. 

an alternative is to check the from_address_status in the view and apply the strong there, keeping html out of the locales. could probably then remove the brakeman issue? 🤔 

up to you both

Reply:
Correct me if I'm wrong here @chrispymm but I'm not sure we can apply the strong in the view with the 'FromAddressPresenter' in it's current state. Currently the whole message is passed into the view as `@presenter.message[:text]`. If we want to apply strong in the view we would have to rethink how to pass that particular message (status: 'pending') in from the presenter as it is only the first half of the message that is bold - I'm happy to give this a shot if we'd like but perhaps we should tackle is in a separate PR?